src/main/scala/org/broadinstitute/dsde/vault/services/analysis/UpdateServiceHandler.scala

```diff
@@ -94,7 +94,7 @@ case class UpdateServiceHandler(requestContext: RequestContext, dmService: Actor
         case true =>
           if (fileCount == bossObjects.size) {
             log.debug("'X-Force-Location' Analysis update complete")
-            requestContext.complete(new Analysis(dmId.get, analysis.input, analysis.metadata, analysis.files))
+            requestContext.complete(new Analysis(dmId.get, analysis.input, analysis.metadata, Option(providedFiles)))
```

Comment:
OK.  So what does the DM return?  Looks like it's the original file paths in both cases.  So we want to leave it alone for the Force case but change it to PUT URLs in the standard case.